db/schema.rb

```diff
@@ -62,6 +62,8 @@
     t.datetime "time_last_updated"
     t.datetime "created_at"
     t.datetime "updated_at"
+    t.integer  "points"
+    t.string   "status",            :default => "incomplete"
```

Comment:
A translation's status is determined dynamically, so keeping this `status` field is unecessary.